src/gui/gui_graphics.hpp

```diff
@@ -65,6 +64,7 @@ struct gfx_object {
 		return (flags & do_transparency_check) != 0;
 	}
 };
+static_assert(sizeof(gfx_object) == 16);
```

Comment:
Are you sure that you need padding for this one? It looks to me like it should be 2 byte aligned, and thus that 10 bytes will fill it completely. If it isn't 2 byte aligned, try just forcing its alignment to that